packages/contentstack-import/package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@contentstack/cli-cm-import",
   "description": "Contentstack CLI plugin to import content into stack",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@oclif/config": "^1.18.3",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K we can remove this package if not used directly

Reply:
@antonyagustine removed @oclif/core dependency from package.json